skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
bump @chibzu @aldigun, still want to get this out. how do you feel about my previous comment?